optaplanner-examples/src/test/java/org/optaplanner/examples/conferencescheduling/solver/ConferenceSchedulingScoreSoftConstraintTest.java

```diff
@@ -32,26 +33,8 @@ public void themeConflict() {
         String theme2 = "theme2";
         String theme3 = "theme3";
         String theme4 = "theme4";
-        Talk talk1 = new Talk(1L)
```

Comment:
We don't need every test twice. Ideally, we want to migrate each of these tests in the xlsx file (presuming they don't exist there already, then we simply want to remove them here).

Reply:
@ge0ffrey https://issues.jboss.org/browse/PLANNER-1232